locales/en/openbridge.json

```diff
@@ -53,6 +53,10 @@
     "deleteFilter": "Delete filter",
     "filters": "Filters",
     "generalInformation": "General information",
+    "loadingProcessor": "Loading processor",
+    "loadingProcessorType": "Loading processor type",
```

Comment:
What are the steps user should be able to see this message please? Once I clicked on a processor in the instance detail page, the screen turn to grey (it was clear to me loading is happening), but no message about it appears. Is that expected?

Reply:
Yes, it is the expected behavior. 

They are screen reader descriptions passed to PF Skeleton component (like [here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/blob/0a82751c5ab0547dbaa065660659d1123b590683/src/app/components/PageHeaderSkeleton/PageHeaderSkeleton.tsx#L44)), so they will be read  only by screen reader users.